config/config.exs

```diff
@@ -18,6 +18,21 @@ config :logger, :console,
   discard_threshold: 2000,
   metadata: [:module, :function, :request_id]
 
+config :logger,
+  backends: [Sentry.LoggerBackend, :console]
+
+config :sentry,
```

Comment:
Just remembered... have you found any docs how Sentry works with a elixir release?

Reply:
@InoMurko nope I haven't looked for any so far. Can follow up with another PR to improve this integration.